temoa/extensions/stochastics/scenario_creator.py

```
@@ -0,0 +1,110 @@
+        raise ValueError("scenario_creator requires 'temoa_config' and 'stoch_config' in kwargs")
+
+    temoa_config: TemoaConfig = kwargs['temoa_config']
+    stoch_config: StochasticConfig = kwargs['stoch_config']
+
+    # 1. Load base data
+    try:
+        with sqlite3.connect(temoa_config.input_database) as con:
+            hybrid_loader = HybridLoader(db_connection=con, config=temoa_config)
+            data_dict = hybrid_loader.create_data_dict(myopic_index=None)
+    except Exception as e:
+        logger.exception("Failed to connect to database %s: %s", temoa_config.input_database, e)
```

Ruff (0.14.10)

Redundant exception object included in `logging.exception` call

(TRY401)